mix.exs

```diff
@@ -0,0 +1,53 @@
+defmodule TechForGoodNearYou.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :tech_for_good_near_you,
+     version: "0.0.1",
+     elixir: "~> 1.4",
+     elixirc_paths: elixirc_paths(Mix.env),
+     compilers: [:phoenix, :gettext] ++ Mix.compilers,
+     start_permanent: Mix.env == :prod,
+     aliases: aliases(),
+     deps: deps()]
+  end
+
+  # Configuration for the OTP application.
+  #
+  # Type `mix help compile.app` for more information.
+  def application do
+    [mod: {TechForGoodNearYou.Application, []},
+     extra_applications: [:logger, :runtime_tools, :httpotion]]
+  end
+
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_),     do: ["lib"]
+
+  # Specifies your project dependencies.
+  #
+  # Type `mix help deps` for examples and options.
+  defp deps do
+    [{:phoenix, "~> 1.3.0-rc"},
+     {:phoenix_pubsub, "~> 1.0"},
+     {:phoenix_ecto, "~> 3.2"},
+     {:postgrex, ">= 0.0.0"},
+     {:phoenix_html, "~> 2.6"},
+     {:phoenix_live_reload, "~> 1.0", only: :dev},
+     {:gettext, "~> 0.11"},
+     {:cowboy, "~> 1.0"},
+     {:httpotion, "~> 3.0.2"}]
```

Comment:
there seems there's a consensus in the community on `HTTPoison` vs `HTTPotion`, I think we should use `HTTPoison` instead

https://elixirforum.com/t/httpoison-vs-httpotion/2221/10